### PR TITLE
jellyfin-mpv-shim: Add version 2.0.2

### PR DESCRIPTION
**Pull request by versedwildcat**
Created a manifest for [Jellyfin MPV Shim](https://github.com/jellyfin/jellyfin-mpv-shim)

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by versedwildcat** [comment]
Well... probably because it's late here, but I can't for the life of me figure out what's wrong with the manifest. Help appreciated

File: bucket/jellyfin-mpv-shim.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "2.0.2",
+    "description": "Jellyfin MPV Shim is a fully featured, cross-platform desktop and cast client for Jellyfin.",
```
Comment (issaclin32) [inline bucket/jellyfin-mpv-shim.json:3]:
```suggestion
    "description": "A fully featured, cross-platform desktop and cast client for Jellyfin",
```

**Comment by issaclin32** [comment]
Persist is not required because config file is at `$Env:AppData\jellyfin-mpv-shim`
(This is a reminder, no need to modify anything)

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**